frontend/libs/studio-components/src/components/StudioPageHeader/StudioPageHeaderLeft/DigdirLogoLink/DigdirLogoLink.tsx

```diff
@@ -16,10 +16,8 @@ export const DigdirLogoLink = ({ title, showTitle }: DigdirLogoLinkProps): React
 
   return (
     <div className={classes.wrapper}>
-      <StudioPageHeaderHeaderButton asChild color={color} variant={variant}>
-        <a href='/' title={title}>
-          <DigdirLogo />
-        </a>
+      <StudioPageHeaderHeaderButton as='a' color={color} variant={variant} href='/' title={title}>
```

Comment:
Could the `as` prop be set inside the `StudioPageHeaderHeaderButton`? I guess all the `StudioPageHeaderHeaderButton`s are suppose to act as `a` elements

Reply:
I dont' think we should make such assumptions. Better to keep the component simple. When it has the name `Button`, it is clear that it renders a button, unless it is explicitly overridden.